pkg/cmd/openshift-sdn-node/kube_proxy.go

```diff
@@ -187,7 +163,7 @@ func newProxyServer(config *kubeproxyconfig.KubeProxyConfiguration, client clien
 		EnableProfiling:    config.EnableProfiling,
 		ConfigSyncPeriod:   config.ConfigSyncPeriod.Duration,
 		HealthzServer:      healthzServer,
-		UseEndpointSlices:  useEndpointSlices,
+		UseEndpointSlices:  true,
```

Comment:
we don't need this anymore as well as the check

Reply:
yeah... I was consider that to be part of the "sync to 1.26" commit (which isn't in this PR) rather than the "stop supporting userspace" commit...